dump/dist/pvm_dump_offload.service.in

```diff
@@ -1,11 +1,11 @@
 [Unit]
-Description=PowerVM Handler
+Description=PowerVM Handler %i
 Wants=phosphor-debug-collector.service
+Wants=obmc-host-started@%i.target
 After=phosphor-debug-collector.service
```

Comment:
phosphor-debug-collector.service is not a valid service.  May be you want to replace with xyz.openbmc_project.Dump.Manager.service and host dump services /  incase you want add service start sequence. or need to play with depends, in the recipes.

Reply:
good catch, i had copied it from some other service